snapshot/git/gitdb/stream.go

```diff
@@ -0,0 +1,66 @@
+package gitdb
+
+import (
+	"fmt"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// A Stream is a sequence of SnapshotWithHistory's that updates via a
```

Comment:
Let's discuss the naming in the design doc (and punt on it here for now; it's easy to rename).

It's a sequence because it advances outside of ScootDB. If a user passes an ID in the Local Scheme and it's not present in the data repo, ScootDB can't get it. With the scheme stream, it knows to fetch.